docs/layers/text-layer.md

```diff
@@ -149,6 +149,18 @@ Options:
 
 `radius` and `cutoff` will be applied only when `sdf` enabled.
 
+##### `wordBreak` (String, optional)
+
+* Default: `null`
+
+CSS `word-break`. `width` has to be provided to use `wordBreak`.
+
+##### `width` (Number, optional)
+
+* Default: `null`
+
+`width` is ignored when `wordBreak` is not provided.
```

Comment:
Is there a reason to require both to be specified explicitly? I would think not providing `width` is enough indication that no wrapping should be performed.

Reply:
I actually want to say if the user would like to use auto wrapping feature, the user has to provide both `width` and `wordBreak`.

Reply:
`wordBreak` can have a default value. It doesn't make sense if someone specifies the max width or a paragraph but does not want wrapping.